### PR TITLE
Ref #97: Replace built-in shadowed str with input_string

### PR DESCRIPTION
**Pull request by flamableconcrete**
## Description
**Addresses** #97

This isn't a full fix for issue 97, but a first stab at it. The issue has more details, but the quick synopsis is that several of the string examples were shadowing the Python built-in function `str`. That being said, it is also related to Pull Request #127 where just a couple days ago one of the snippets was re-written this way.

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. 
- [ ] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by flamableconcrete** [comment]
Hopefully that looks better than `input_string`!

File: snippets/byte_size.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ tags: string, beginner
 
 Returns the length of a string in bytes.
 
-Use `string.encode('utf-8')` to encode the given string and return its length.
+Use `s.encode('utf-8')` to encode the given string and return its length.
```
Comment (Chalarangelo) [inline snippets/byte_size.md]:
```suggestion
Use `str.encode('utf-8')` to encode the given string and return its length.
```

File: snippets/capitalize_every_word.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ tags: string,beginner
 
 Capitalizes the first letter of every word in a string.
 
-Use `string.title()` to capitalize first letter of every word in the string.
+Use `s.title()` to capitalize first letter of every word in the string.
```
Comment (Chalarangelo) [inline snippets/capitalize_every_word.md]:
```suggestion
Use `str.title()` to capitalize first letter of every word in the string.
```

File: snippets/is_anagram.md [hunk 3]
```diff
@@ -5,13 +5,13 @@ tags: string,intermediate
 
 Checks if a string is an anagram of another string (case-insensitive, ignores spaces, punctuation and special characters).
 
-Use `str.replace()` to remove spaces from both strings.
+Use `s.replace()` to remove spaces from both strings.
```
Comment (Chalarangelo) [inline snippets/is_anagram.md]:
```suggestion
Use `str.replace()` to remove spaces from both strings.
```

File: snippets/palindrome.md [hunk 4]
```diff
@@ -5,14 +5,14 @@ tags: string,intermediate
 
 Returns `True` if the given string is a palindrome, `False` otherwise.
 
-Use `str.lower()` and `re.sub()` to convert to lowercase and  remove non-alphanumeric characters from the given string. 
+Use `s.lower()` and `re.sub()` to convert to lowercase and  remove non-alphanumeric characters from the given string.
```
Comment (Chalarangelo) [inline snippets/palindrome.md]:
```suggestion
Use `str.lower()` and `re.sub()` to convert to lowercase and  remove non-alphanumeric characters from the given string. 
```

**Pull request merged**